python/lib/cloudutils/serviceConfig.py

```diff
@@ -14,38 +14,44 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from utilities import writeProgressBar, bash
-from cloudException import CloudRuntimeException, CloudInternalException, formatExceptionInfo
+from .utilities import writeProgressBar, bash
+from .cloudException import CloudRuntimeException, CloudInternalException, formatExceptionInfo
 import logging
-from networkConfig import networkConfig
+from .networkConfig import networkConfig
 import re
-from configFileOps import configFileOps
+from .configFileOps import configFileOps
 import os
 import shutil
 
 # exit() error constants
 Unknown = 0
 CentOS6 = 1
 CentOS7 = 2
-Ubuntu = 3
-RHEL6 = 4
-RHEL7 = 5
+CentOS8 = 3
```

Comment:
Does changing these constants cause any error?

Reply:
@rhtyd have checked the code, as per my understanding, value these constants is used within this file only at line 533 cc @davidjumani